code-challenges.test.js

```diff
@@ -46,6 +83,25 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
-// b) Create the function that makes the test pass.
+describe("someNouns", () => {
+    it("takes in array of words and returns array with all words capatalized", () => {
+        expect(someNouns(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect(someNouns(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+    })
+})
 
-// Pseudo code:
\ No newline at end of file
+// b) Create the function that makes the test pass.
+// create a function that takes both arrays randomNouns 1&2
+const someNouns = (strings) => {
+   // iterate usingn strings .map 
+    return strings.map((value) => {
+        //using bracket notation 0 we .uppercase the first letter in the string and slice 1 returns all the letters after the 1st capatilized letter
+        return value[0].toUpperCase() + value.slice(1);
+    });
+}
```

Comment:
When you name your parameter, I suggest using what data you're working with. The function is taking in an array, so I would name the parameter `arr` or something like that, instead of strings, because then it might get confusing. You might think you're working with a string when you're working with an array, which may cause issues.